vizro-core/docs/pages/development/contributing.md

```diff
@@ -145,6 +145,23 @@ Vizro's version is given by `__version__` in [`src/vizro/__init__.py`](https://g
 
 To build the source distribution and wheel, run `hatch build`.
 
+### Steps to create a Vizro release
+
+The following steps are required to create a release of Vizro. Note that this is only done by maintainers:
+
+1. Checkout a new branch with a suitable name, e.g. `git checkout -b release/v0.2.4`
```

Comment:
Does it mean that `prep-release` will create this version? I've created branch `release/v1.0.0`, but it created `0.1.1`

Reply:
No, the name is arbitrary, let me adjust the docs then